setup-env

```diff
@@ -27,7 +27,7 @@ export EV3DEV_TOOLCHAIN="/opt/arm-2011.03/bin"
 export EV3DEV_ABI="arm-none-eabi-"
 
 # This is where the scripts put all of generated files
-export EV3DEV_BUILD_AREA="$HOME/build-area"
+export EV3DEV_BUILD_AREA="$(pwd)/build-area"
```

Comment:
I thought so, that you did this on purpose. But the place wasn't very obvious and I didn't think it make much sense to first create a fresh new folder, whereever the user wants, and in the next step a folder is created in $HOME. ;)

I thought also on the $(pwd)/../build-area idea. But even if you stated the parent folder shall be created first, I doubt all users will do this. So again the script would work in a directory, the user may not want to. So imo the .gitignore idea is the best. Furthermore it would be very similar to most cmake build steps:
git clone some_cmake_project; cd some_cmake_project; mkdir build; cd build; cmake ..; make